b2sdk/raw_api.py

```diff
@@ -281,6 +292,7 @@ class B2RawApi(AbstractRawApi):
 
     def __init__(self, b2_http):
         self.b2_http = b2_http
+        self.logger = getLogger(__name__)
```

Comment:
Here you would potentially have multiple loggers with the same name - I'm not sure if this is a good idea. If we'd initialize multiple classes in this file, they would all need to run getLogger in their constructors to get access to the logger, keep a reference to it, a module-level function wouldn't have anywhere to keep it in state... That is why in `b2sdk` by convention we use module-scoped loggers that we define right after imports as `logger = logging.getLogger(__name__)`. If it is not a big problem, please follow that convention.

Reply:
Ah good. I hate this convention too. I copied it from somewhere in the project because I thought that was your convention, but it looks like that was from my `.venv` folder :P